storage/registers.go

```diff
@@ -0,0 +1,33 @@
+package storage
+
+import (
+	"io"
+
+	"github.com/onflow/flow-go/model/flow"
+)
+
+// Registers defines methods for the register index.
+type Registers interface {
+	RegisterReader
+	RegisterWriter
+	io.Closer
+}
+
+// RegisterReader defines read-only operations on the register index.
+type RegisterReader interface {
+	// Get register by the register ID at a given block height.
+	//
+	// If the register at the given height was not indexed, returns the highest
+	// height the register was indexed at.
+	// An error is returned if the register was not indexed at all. Expected errors:
+	// - storage.ErrNotFound if the register was not found in the db
+	Get(ID flow.RegisterID, height uint64) (flow.RegisterValue, error)
+}
```

Comment:
I think we should add this method to return the latest height
```suggestion
	// latest finalized and executed height
	Latest() (height uint64) 
}
```